portfolio/src/main/webapp/index.html

```diff
@@ -13,47 +16,67 @@ <h1>Chidera Wokonko</h1>
             <div class="topnav-right">
                 <a href="index.html">About</a>
                 <a href="Proj.html">Projects</a>
-                <a href="Chidera_Wokonko_Resume.pdf">Resume</a>
-                <a href="#">Contact</a>
+                <a href="Chidera_Wokonko_Resume.pdf" target="_blank">Resume</a>
+                <a href="#connect">Contact</a>
             </div>
         </div>
       <p>Welcome to my page!</p>
-      <img src = "images/Chidera-12.jpg" width=50% height=90%/>
-      <p> My name is Chidera Wokonko and I'm currently an undergraduate student at Cornell University majoring in Electrical and Computer Engineering (ECE) with a minor in robotics. 
-          Pursuing an engineering degree is no easy task, so to make that transition easier for subsequent classes I serve as a CUempower mentor for a group of freshman students 
-          who are part of Diversity Programs in Engineering at Cornell University. When I'm not doing problem sets or studying for an exam I'm working out, watching movies, implementing
-          my personal IOT devices or enjoying quality time with friends. I run a fitness vlog on Instagram called COFITNESS and I host weekly conversation hours at my university for 
-          people who want to learn the Nigerian native tongue--Igbo. 
-       </p>
-       <p>.......................................................................</p>
-       <p> My favorite thing about being an ECE major is the opportunity I have to navigate the boundary between the 
-           hardware and software realm. I'm really passionate about physical computing as it enables me to blur the 
-           lines between the digital and physical world. Check out my projects to see how I've put passion to practice.
-           <ul>
-				<li><a href="Proj.html" class="projects" style="text-decoration:none" >Projects</a></li>
-			</ul>
-        </p>
-       <h3>Fast Facts</h3>
-            <ul>
-                <li> <button class="fact1" onclick="ShowFact(1)">Languages</button> </li>
+      <div>
+          <p class="hidden"> ................................................................................ <a class="glow">Touch the Picture!</a></p>
+      </div>
+        <div class="flip-card">
+            <div class="flip-card-inner">
+        <div class="flip-card-front">
+            <img src="images/Chidera-12.jpg" alt="Avatar" style="width:337px;height:502px">
+        </div>
+        <div class="flip-card-back">
+            <h1>Chidera Wokonko</h1> 
+            <p>Co-Founder, COFITNESS</p>
+            <p>Fitness Enthusiast X Engineer</p> 
+            <p></p>
+            <div class="icons">
+                <a href="https://github.com/Cwokonko/my-portfolio.git" style="text-decoration:none"><i class="fa fa-github" style="color: white;"></i></a> 
```

Comment:
I recommend making giving all links the `target="_blank"` attribute. That way, when users click on a link, they'll open in a new tab.

Reply:
I added it now